plugins/wallabag.koplugin/main.lua

```diff
@@ -258,6 +262,17 @@ function Wallabag:addToMainMenu(menu_items)
                             },
                         },
                     },
+                    {
+                        text = _("Send review as tags"),
```

Comment:
I think a little bit more explanation would be helpful. :-) Something like this:
```suggestion
                        text = _("Send review as tags"),
                        help_text = _("This allow you to write tags in the review field, separated by commas, which can then be sent to Wallabag."),
```